File: C/Program Menghitung Luas Permukaan Balok.c
#include <stdio.h>

int main() {
    int panjang, lebar, tinggi, luas_permukaan;

    printf("Program Menghitung Luas Permukaan Balok\n");
    printf("visit us hobingoding.com\n\n");
    // Input Panjang
    printf("Input Panjang : ");
    scanf("%d", &panjang);
    // Input Lebar
    printf("Input Lebar : ");
    scanf("%d", &lebar);
    // Input Tinggi
    printf("Input Tinggi : ");
    scanf("%d", &tinggi);

    // Hitung Luas Permukaan
    luas_permukaan = 2 * ((panjang * lebar) + (panjang * tinggi) + (lebar * tinggi));
    printf("Luas Permukaan Balok : %d", luas_permukaan);
    return 0;
}

File: Python/Program Menghitung Luas Permukaan Balok.py
print("Program Menghitung Luas Permukaan Balok")
print("visit us hobingoding.com")
print()
# Input Panjang
panjang = int(input("Input Panjang : "))
# Input Lebar
lebar = int(input("Input Lebar : "))
# Input Tinggi
tinggi = int(input("Input Tinggi : "))

# Hitung Luas Permukaan
luas_permukaan = 2 * ((panjang * lebar) + (panjang * tinggi) + (lebar * tinggi))
print("Luas Permukaan Balok :", luas_permukaan)